src/glyphs/devicons/mapping

```diff
@@ -51,7 +51,7 @@ E62D E72D vorillaz/smashing_magazine.svg smashing_magazine
 E62E E72E apachekafka/apachekafka-original.svg apachekafka
 E62F E72F apachespark/apachespark-original.svg apachespark
 E630 E730 apl/apl-plain.svg apl
-E631 E731 appwrite/appwrite-plain.svg appwrite
+E631 E731 appwrite/appwrite-original.svg appwrite
```

Comment:
There are more such changes, substituting `original` for `plain`.

Did you do that on purpose, of came it out of the analyzer script? :thinking: 

Thanks for the PR, btw. :green_heart:

Reply:
> Did you do that on purpose, of came it out of the analyzer script

This was on purpose as some `plain` versions were removed/renamed in v2.17.0. For example, `appwrite` was renamed to `original` here https://github.com/devicons/devicon/pull/2196/files#diff-8fb0089a81b777eae04d6ac8f37becd0a07c41dd808f17e9e90eafb7411a8f60

Reply:
Ah I see, thanks!

It is unlikely that I have time for an in-depth look before August, sorry.
But your PR is very welcome! It should not take long to review, maybe I can squeeze it in somehow.
At the moment I work from 6 am to 8 pm every day :-(